packages/browser/src/node/providers/webdriver.ts

```diff
@@ -92,17 +103,15 @@ export class WebdriverBrowserProvider implements BrowserProvider {
     return capabilities
   }
 
-  async openPage(_contextId: string, url: string) {
-    const browserInstance = await this.openBrowser()
+  async openPage(contextId: string, url: string) {
+    const browserInstance = await this.openBrowser(contextId)
     await browserInstance.url(url)
   }
 
   async close() {
-    await Promise.all([
-      this.browser?.sessionId ? this.browser?.deleteSession?.() : null,
-    ])
-    // TODO: right now process can only exit with timeout, if we use browser
-    // needs investigating
-    process.exit()
```

Comment:
Could we now following from `test-utils`? 

https://github.com/vitest-dev/vitest/blob/2da57790e318ec900ba6df55e1ffe552419dca05/test/test-utils/index.ts#L17-L19

Reply:
Yes